dashboard/15-final/app/ui/acme-logo.tsx

```diff
@@ -1,29 +1,13 @@
-          clipRule="evenodd"
-        />
-      </g>
-      <path
-        fill="#fff"
-        d="m36.808 29.372-3.328.128-.32-1.472h.192c.981-.107 1.728-.373 2.24-.8.533-.448.981-1.173 1.344-2.176L43.72 6.78h1.536l7.552 18.784c.448 1.045.843 1.728 1.184 2.048.363.32.939.48 1.728.48l.352 1.408a71.114 71.114 0 0 0-4.512-.16c-1.323 0-2.613.053-3.872.16l-.288-1.376h.128c1.685-.021 2.528-.32 2.528-.896 0-.277-.768-2.176-2.304-5.696h-7.968l-1.568 4.352a2.82 2.82 0 0 0-.096.672c0 .448.203.79.608 1.024.405.235 1.099.384 2.08.448l.384 1.472-4.384-.128Zm6.656-17.952-3.072 8.448h6.624l-3.552-8.448Zm23.955 6.912c-.363-.363-.864-.725-1.504-1.088-.64-.363-1.376-.544-2.208-.544-.832 0-1.6.459-2.304 1.376-.704.896-1.056 2.165-1.056 3.808 0 1.643.426 3.061 1.28 4.256.853 1.173 2.122 1.76 3.808 1.76 1.344 0 2.453-.437 3.328-1.312a.217.217 0 0 1 .16-.064c.064 0 .117.021.16.064l.64.736c-1.579 1.707-3.414 2.56-5.504 2.56-2.07 0-3.723-.661-4.96-1.984-1.216-1.323-1.824-2.987-1.824-4.992 0-2.475.682-4.352 2.048-5.632 1.386-1.28 3.114-1.92 5.184-1.92.896 0 1.802.128 2.72.384.938.256 1.642.576 2.112.96a.284.284 0 0 1 .032.128c0 .299-.33.928-.992 1.888a.476.476 0 0 1-.352.128c-.128 0-.384-.17-.768-.512Zm4.652-.64-.48.032-.288-.928c1.493-.363 2.592-.725 3.296-1.088.704-.363 1.099-.565 1.184-.608.277 0 .437.021.48.064.085.085.15.768.192 2.048 1.707-1.237 3.253-1.856 4.64-1.856 1.835 0 3.115.64 3.84 1.92 1.792-1.28 3.456-1.92 4.992-1.92 3.093 0 4.64 1.59 4.64 4.768v7.104c0 .384.096.65.288.8.192.128.661.224 1.408.288l.288 1.28-3.136-.096-3.456.096-.256-1.28c1.024-.085 1.632-.17 1.824-.256.213-.107.32-.384.32-.832v-6.016c0-1.365-.267-2.315-.8-2.848-.533-.555-1.365-.832-2.496-.832-1.11 0-2.165.352-3.168 1.056a7.83 7.83 0 0 1 .16 1.536v7.104c0 .384.096.65.288.8.192.128.661.224 1.408.288l.288 1.28-3.136-.096-3.456.096-.256-1.28c1.024-.085 1.632-.17 1.824-.256.213-.107.32-.384.32-.832v-6.016c0-1.365-.267-2.325-.8-2.88-.512-.555-1.28-.832-2.304-.832-1.024 0-2.09.299-3.2.896v8.832c0 .384.096.65.288.8.192.128.661.224 1.408.288l.288 1.28-3.136-.096-3.456.096-.256-1.28c1.024-.085 1.632-.17 1.824-.256.213-.107.32-.384.32-.832V19.74c0-.768-.139-1.301-.416-1.6-.277-.299-.715-.448-1.312-.448Zm38.587 4.032h-9.28v.416c0 1.557.437 2.912 1.312 4.064.875 1.13 2.037 1.696 3.488 1.696s2.645-.47 3.584-1.408c.064-.064.117-.096.16-.096s.096.032.16.096l.704.8c-1.6 1.728-3.456 2.592-5.568 2.592-2.091 0-3.744-.661-4.96-1.984-1.195-1.323-1.792-3.083-1.792-5.28 0-2.219.683-3.979 2.048-5.28 1.365-1.323 2.923-1.984 4.672-1.984 1.749 0 3.125.533 4.128 1.6 1.003 1.045 1.504 2.517 1.504 4.416 0 .235-.053.352-.16.352Zm-3.04-1.504c0-2.347-.907-3.52-2.72-3.52-.96 0-1.739.341-2.336 1.024-.597.661-.96 1.493-1.088 2.496h6.144Z"
-      />
-      <defs>
-        <clipPath id="globe-logo">
-          <path fill="#fff" d="M0 4h28v28H0z" />
-        </clipPath>
-      </defs>
-    </svg>
+      <GlobeAltIcon className="h-12 w-12 rotate-[15deg]" />
+      <p className="text-[44px] ">Acme</p>
+    </div>
```

Comment:
Logo could be dramatically simplified using a Heroicon and rotating it just a touch. :chef-kiss:

Reply:
:chef-kiss: